core/shared/src/main/scala/zio/internal/ZMetrics.scala

```diff
@@ -0,0 +1,32 @@
+package zio.internal
+
+import zio.internal.metrics._
+
+trait ZMetrics {
```

Comment:
All this definitely belongs in the internal package as hopefully users won't see or touch it. 👍